dashboard/test/helpers/workshop_survey_results_helper_test.rb

```diff
@@ -107,6 +107,23 @@ class Pd::WorkshopSurveyResultsHelperTest < ActionView::TestCase
     assert_equal 3.67, result_hash[:how_clearly_presented]
   end
 
+  test 'averaging across multiple surveys' do
+    workshop_1 = create(:pd_workshop, :local_summer_workshop, num_sessions: 1, num_facilitators: 2, num_completed_surveys: 5)
+    workshop_2 = create(:pd_workshop, :local_summer_workshop, num_sessions: 1, num_facilitators: 3, num_completed_surveys: 10)
+
+    workshop_2.survey_responses.each do |response|
+      response.update_form_data_hash({how_clearly_presented: {workshop_2.facilitators.first.name => 'Not at all clearly', workshop_2.facilitators.second.name => 'Not at all clearly', workshop_2.facilitators.third.name => 'Not at all clearly'}})
+      response.save
+    end
+
+    # With 10 people saying "Not at all clearly" to 3 facilitators, and 5 people saying
+    # "Extremely Clearly" to 2 facilitators, we'd expect the answer to be
+    # [(10 * 3 * 1) + (5 * 2 * 5)] / (10 + 30)
```

Comment:
It makes sense, but it seems to appear arbitrarily in the middle of the test case. I'd expect it to inform one of the expected results in an assertion. Does it inform the 2 on line 124? Maybe add `= 2` and drop the empty line on line 122 so this is more clearly related.